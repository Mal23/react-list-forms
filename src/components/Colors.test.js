import React from 'react';
import { shallow } from 'enzyme';
import Colors from './Colors';

describe('Colors component', () => {
  it('renders Colors', () => {
    const colors = [
      { name: 'Red', rgb: { red: 255, green: 0, blue: 0 } },
      { name: 'Blue', rgb: { red: 0, green: 0, blue: 255 } },
      { name: 'Green', rgb: { red: 0, green:255, blue: 0 } }
    ];
    const wrapper = shallow(<Colors colors={colors} />);
    expect(wrapper).toMatchSnapshot();
  });
});
