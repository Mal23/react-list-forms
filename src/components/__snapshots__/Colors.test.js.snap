// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Colors component renders Colors 1`] = `
<ul>
  <li
    key="Red"
  >
    <Color
      name="Red"
      rgb={
        Object {
          "blue": 0,
          "green": 0,
          "red": 255,
        }
      }
    />
  </li>
  <li
    key="Blue"
  >
    <Color
      name="Blue"
      rgb={
        Object {
          "blue": 255,
          "green": 0,
          "red": 0,
        }
      }
    />
  </li>
  <li
    key="Green"
  >
    <Color
      name="Green"
      rgb={
        Object {
          "blue": 0,
          "green": 255,
          "red": 0,
        }
      }
    />
  </li>
</ul>
`;
